ci/build_image.sh

```diff
@@ -1,42 +1,30 @@
 #!/usr/bin/env bash
 set -euxo pipefail
 
-source $(dirname $0)/env.sh
-
 print_help() {
     echo "Usage: ci/build_image.sh <build|push> <build|release>"
     exit 1
 }
 
-case "${2:-}" in
-    build)
-        DOCKERFILE="$SCRIPT_DIR/docker/build/Dockerfile"
-        IMAGE_TAG="${CONTAINER_REGISTRY}/low-level-hacks/automation/nagger/build:${NAGGER_BUILD_TAG}"
-        ARGS=""
-        ;;
-    release)
-        DOCKERFILE="$SCRIPT_DIR/docker/release/Dockerfile"
-        IMAGE_TAG="${CONTAINER_REGISTRY}/low-level-hacks/automation/nagger/release:${CI_COMMIT_TAG}"
-        ARGS="--build-arg CONTAINER_REGISTRY=${CONTAINER_REGISTRY} \
-              --build-arg NAGGER_BUILD_TAG=${NAGGER_BUILD_TAG}"
-        ;;
-    *)
-        echo "Unrecognized image '${2:-}'"
-        print_help
-esac
+SCRIPT_DIR="$PWD/ci"
+BASE_DIR=$PWD
+# Github pull request has github.ref_name (which is used to set this) in
+# format "${pull_request_id}/merge" and '/' is not a valid docker tag so
+# we convert it to '_'.
+CI_COMMIT_TAG=$(echo $CI_COMMIT_TAG | tr '/' '_')
+
+DOCKERFILE="$SCRIPT_DIR/docker/release/Dockerfile"
+IMAGE_TAG="${CONTAINER_REGISTRY}/nordsecurity/nagger:${CI_COMMIT_TAG}"
+ARGS="--build-arg CONTAINER_REGISTRY=${CONTAINER_REGISTRY}"
 
 case "${1:-}" in
     build)
         docker build -f "$DOCKERFILE" -t "$IMAGE_TAG" ${BASE_DIR} ${ARGS}
         ;;
     push)
-        if $(docker manifest inspect $IMAGE_TAG &> /dev/null); then
```

Comment:
Why is this removed ?

Reply:
This is removed to make it possible to push updated image version for new commits on the branch (so that the author of the branch can use that image for testing). I can imagine that this guard to not override images was useful when we worked with the images manually but since we move all the work to automated gh jobs there is no reason to worry that a released tag will be overwritten manually by accident since there is no reason to do anything like that locally.